hash/risc0/bench/methods/guest/src/bin/sha256_accelerated.rs

```diff
@@ -0,0 +1,21 @@
+
+// accelerated sha2 crate
```

Comment:
the sha256 and sha256_accelerated benchmarks appears to take approximately same amount of time for any input size.

so I think either both is accelerated or neither. If both is accelerated, then it appears to be very slow (on my machine about 85 seconds for 4096 bytes. I estimate that a naive circom implementation would be about 10x faster!)